index.html

```diff
@@ -276,7 +276,7 @@ <h4 class="panel-title">
             </li>
           </ul>
         </div>
-        <table ng-class="{'notimestamp':!settings.showtimestamp,'notimestampseconds':!settings.showtimestampSeconds}">
+        <table>
```

Comment:
I would suggest you keep this around and use classes on the `span`s to show/hide instead of `ng-if`s